File: ez.js
let abc = 0
const def = 69

console.log(abc)

abc = abc + 1

console.log(abc)

console.log(def)

abc = 0
console.log("Activating Loop")
while (abc <= 50) {
    if (abc > 25) {
        console.log(abc)
    } else {
        console.log("Number hidden until 25")
    }
    abc += 1
}

File: ez.lua
local abc = 0
local def = 69

print(abc)

abc = abc + 1

print(abc)

print(def)

abc = 0
print("Activating Loop")
while abc <= 50 do
    if abc > 25 then
        print(abc)
    else
        print("Number hidden until 25")
    end

    abc = abc + 1
end
